datanode/sqlstore/migrations/0010_live_liquidity_provisions.sql

```diff
@@ -0,0 +1,57 @@
+-- +goose Up
+
+create table live_liquidity_provisions
```

Comment:
probably my misunderstanding, but I ee this table has been added, but does not seem it is used anywhere?  I would have expected to see a change to the corresponding sqlstore for liquidity provisions?